spec/std/yaml/serialization_spec.cr

```diff
@@ -379,6 +382,10 @@ describe "YAML serialization" do
 
     it "does for Enum" do
       YAMLSpecEnum.from_yaml(YAMLSpecEnum::One.to_yaml).should eq(YAMLSpecEnum::One)
+      {"--- one\n", "--- one\n...\n"}.should contain(YAMLSpecEnum::One.to_yaml)
```

Comment:
There's a helper in this file to specifically deal with different behaviour of the YAML library.
```suggestion
      assert_yaml_document_end(YAMLSpecEnum::One.to_yaml, "--- one\n")
```